Organic/GraphQL.php

```diff
@@ -143,38 +174,48 @@ public function global_ads_config() {
                 'type' => 'OrganicConfig',
                 'description' => __( 'Sitewide Configuration for Organic Platform', 'organic' ),
                 'resolve' => function() {
-                    $connatix = $this->organic->getConnatixConfig();
-                    $testEnabled = $this->organic->getOrganicPixelTestPercent() < 100 &&
-                        $this->organic->getOrganicPixelTestPercent() > 0;
+                    $testEnabled = $this->organic->useSplitTest();
                     return [
-                        'adsEnabled' => $this->organic->isEnabled(),
```

Comment:
This function is called `global_ads_config`, but maybe we should rename it since it seems to be general Organic config?